### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by antocommi**
First ever pull request and commit!

### CODE REVIEW

**Comment by Sam-damn** [comment]
@antocommi  just for future reference u should be performing your pull request on the dev branch and not main , the main branch should only be updated when new releases are ready to be published , that is in your forked repository u commit changes to the dev branch in your fork and then you start the pull request against the Milind220:main , if you want more details you can check the discussion tab it has information on the branching model that is used in this repository

**Comment by Milind220** [comment]
Hey @antocommi , I've changed the target branch of your PR from `main` to `dev`. This is because, as @Sam-damn pointed out, any commits to the `main` branch is automatically considered a new release. Instead we use the `dev` branch to make new changes, which makes sure that `main` is always stable.

Thanks for your contribution! I'll just review it :)

File: src/ozone/ozone.py [hunk 1]
```diff
@@ -213,6 +245,30 @@ def get_city_air(
...
+            locations (list): A list of pair (latitude,longitude) to get data for.
+            data_format (str): File format. Defaults to 'df'. Choose from 'csv', 'json', 'xslx'.
+
+        Returns:
+            pandas.DataFrame: The dataframe containing the data. (If you
+            selected another data format, this dataframe will be empty)
+        """
+        for loc in locations:
+            # This just makes sure that it's always a returns a pd.DataFrame. Makes mypy happy.
+            df = pandas.DataFrame(self.get_lat_lon_air(loc[0], loc[1], df=df, params=params))
```
Comment (Milind220) [inline src/ozone/ozone.py:267]: I think you forgot to rename the `get_lat_long_air` method here. Rename to `get_coordinate_air`

**Review by Milind220 — CHANGES_REQUESTED** [review]
I've left 3 change requests, all regarding the name of the methods. Well done though! everything works as expected :)

**Comment by antocommi** [comment]
Renamed the methods and changed the method reference

**Comment by Milind220** [comment]
@antocommi Awesome! thanks again for the contribution 💯

**Review by Milind220 — APPROVED** [review]

**Pull request merged**